sdk/storage/Azure.ResourceManager.Storage/tests/Tests/BlobContainerTests.cs

```diff
@@ -215,7 +215,7 @@ public async Task ExtendImmutabilityPolicy()
 
             //create immutability policy
             ImmutabilityPolicyData immutabilityPolicyModel = new ImmutabilityPolicyData() { ImmutabilityPeriodSinceCreationInDays = 3 };
-            ImmutabilityPolicy immutabilityPolicy = (await container.GetImmutabilityPolicy().CreateOrUpdateAsync(true, ifMatch: "", parameters: immutabilityPolicyModel)).Value;
```

Comment:
passing an empty string for ifMatch was resulting in inconsistent behavior between platforms - in net core 3.1, the header is omitted, while in other frameworks it is left as is. Since empty string isn't a valid value, I am going to try removing it.
/cc @archerzz

Reply:
Why was this passing before?

Reply:
It seems a regression in .net tech stack. According to this [article](https://thomaslevesque.com/2020/07/23/aspnet-core-iis-and-empty-http-headers/#:~:text=HTTP%20headers%20are%20key%2Fvalue%20pairs%20sent%20at%20the,or%20the%20absence%20of%20that%20header%20are%20equivalent.), there is an ugly workaround, but that doesn't apply here. 
I think we can remove `ifMatch` here. `IfMatch: ""` was migrated from track1 test case which we didn't take close look at.

Reply:
Interesting! Thanks for finding that @archerzz!